src/node.cc

```diff
@@ -80,6 +80,7 @@
 #include <stdlib.h>
 #include <string.h>
 #include <sys/types.h>
+#include <fcntl.h>
```

Comment:
Can you please maintain the alphabetical order? A comment detailing what macro is used like above for `PATH_MAX` would be helpful too.